website/src/components/genspectrum/GsTextInput.tsx

```diff
@@ -40,7 +40,7 @@ export function GsTextInput<LapisField extends string>({
             lapisField={lapisField}
             placeholderText={placeholderText}
             width={width}
-            initialValue={initialValue ?? ''}
+            value={value ?? ''}
```

Comment:
Do we need it to be defined or was this something of the web-components issues?
```suggestion
            value={value}
```